test/e2e/handler/rollback_test.go

```diff
@@ -75,17 +137,77 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			resetDesiredStateForNodes()
 		})
 		It("[test_id:3793]should rollback to a good gw configuration", func() {
+			By("Should not be available") // Fail fast
+			policyConditionsStatusConsistently().ShouldNot(containPolicyAvailable())
+
 			By("Wait for reconcile to fail")
 			waitForDegradedTestPolicy()
 			By(fmt.Sprintf("Check that %s is rolled back", primaryNic))
 			Eventually(func() bool {
-				return dhcpFlag(nodes[0], primaryNic)
+				return autoDNS(nodes[0], primaryNic)
 			}, 480*time.Second, ReadInterval).Should(BeTrue(), "DHCP flag hasn't rollback to true")
```

Comment:
The comment does not match the returned function name. Shouldn't this test keep dhcpFlag?

Reply:
Ups